### PR TITLE
Theming Typeahead autocomplete options

### PR DESCRIPTION
**Pull request by kyleknighted**
## Description

Adding theme to autocomplete options generated by Typeahead components

## Motivation and Context

Fixes Graylog2/graylog-plugin-enterprise-integrations#353

## Screenshots (if appropriate):
image
image

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (non-breaking change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: graylog2-web-interface/src/components/common/TypeAheadFieldInput.jsx [hunk 1]
```diff
@@ -3,14 +3,12 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import Immutable from 'immutable';
 import $ from 'jquery';
-import Typeahead from 'typeahead.js'; // Need to import this to load typeahead, even if the variable is never used
+import 'typeahead.js';
```
Comment (Gaya) [inline graylog2-web-interface/src/components/common/TypeAheadFieldInput.jsx:6]: 👏

File: graylog2-web-interface/src/components/common/TypeAheadFieldInput.jsx [hunk 2]
```diff
@@ -57,22 +65,23 @@ class TypeAheadFieldInput extends React.Component {
...
-            if (this.props.autoFocus) {
+            if (autoFocus) {
               fieldInput.focus();
               fieldInput.typeahead('close');
             }
           },
         );
 
+      // eslint-disable-next-line react/no-find-dom-node
       const fieldFormGroup = ReactDOM.findDOMNode(this.fieldInput);
```
Comment (Gaya) [inline graylog2-web-interface/src/components/common/TypeAheadFieldInput.jsx:76]: Isn't `this.fieldInput` the DOM Node already?

**Review by Gaya — APPROVED** [review]
Looks fine to me

File: graylog2-web-interface/src/components/common/TypeAheadFieldInput.jsx (hunk 2 above)
Comment (kyleknighted) [reply graylog2-web-interface/src/components/common/TypeAheadFieldInput.jsx:76]: I didn't investigate too deeply, was just clearing out lint errors and I didn't want to fully rebuild these components. Also, these changes aren't necessary for the CSS update, so I'm happy to revert them if we feel it makes more sense to just focus on the style updates in this PR.

Comment (Gaya) [reply graylog2-web-interface/src/components/common/TypeAheadFieldInput.jsx:76]: I'd leave it like this. I also ran into old components needing an update thus trigging a bunch of eslint errors. Suppressing is a good option in that case

**Pull request merged**